experimental/packages/otlp-proto-exporter-base/tsconfig.esnext.json

```diff
@@ -1,9 +1,8 @@
 {
   "extends": "../../../tsconfig.base.esnext.json",
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
Why did you remove this directive?

Reply:
Hey, I'm not sure how this change ended up happening. Reverting it.